spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -145,6 +161,11 @@ def setup_toolbar(self):
                 tip=_("Save All Images..."),
                 triggered=self.thumbnails_sb.save_all_figures_as)
 
+        copyfig_btn = create_toolbutton(
+            self, icon=ima.icon('editcopy'),
+            tip=_("Copy Image..."),
```

Comment:
This isn't consistent with the "Copy Image" (no ellipses) as shown in the context menus, which should be preferred here since the universal UI convention is the ``...`` signifies the need to complete another dialog before the action is taken, which is not the case here.

Reply:
I will fix it.

Reply:
Thanks!